packages/react-ui-core/src/List/List.js

```diff
@@ -0,0 +1,49 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import ListItem from './ListItem'
+
+export default class List extends PureComponent {
+  static propTypes = {
+    theme: PropTypes.object,
+    className: PropTypes.string,
+    nodeType: PropTypes.string,
+    Item: PropTypes.node,
+    items: PropTypes.array,
+    listItemNodeType: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+    nodeType: 'ul',
+    listItemNodeType: 'li',
+    Item: ListItem,
+    items: [],
+  }
+
+  render() {
+    const {
+      theme,
+      className,
+      Item,
+      items,
+      listItemNodeType,
+      ...props
+    } = this.props
+
+    return (
+      <this.props.nodeType
```

Comment:
Let's destructure this above.  Reason being is for 2 things:
1. We don't have to look it up twice
2.  It then doesn't get passed down to `<Item />`